tests/unit/connect/command.cpp

```diff
@@ -14,11 +14,11 @@ namespace {
 SharedBuffer buffer;
 
 template <class D>
-D command_test(const string_view cmd) {
+D command_test(char *cmd, size_t size) {
```

Comment:
Wouldn't it be kind of easier to just make a copy of the string here instead of having to add another `char cmd[] =` to each one test?